tests/scanossapi-test.py

```
@@ -0,0 +1,37 @@
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+"""
+
+import unittest
+from scanoss.scanossapi import ScanossApi
+
+class MyTestCase(unittest.TestCase):
+
+    def test_scanoss_generic_headers(self):
+        scanoss_api = ScanossApi(debug=True, req_headers={'x-api-key': '123455', 'generic-header': 'generic-header-value'})
```

Ruff (0.8.2)

Line too long (123 > 120)

(E501)